doc/plugins.rst

```diff
@@ -1,2 +1,147 @@
 Available plugins and configuration
 ===================================
+
+This page lists the available plugins. Note that to enable a plugin, your bot
+app should include an ``enabled = true`` entry in the ``pyproject.toml`` file
+under the section for the specific plugin.
+
+CircleCI Artifacts
+------------------
+
+The CircleCI service provides the option of `storing build artifacts
+<https://circleci.com/docs/2.0/artifacts/>`_. The baldrick plugin will
+automatically post the link to the artifacts as a status check in a GitHub pull
+request to avoid having to click through multiple pages to find the link to the
+artifacts. To enable this plugin, include the following in your
+``pyproject.toml`` file::
+
+    [ tool.<your-bot-name>.circleci_artifacts ]
+    enabled = true
+
+There are no further configuration options for this plugin.
```

Comment:
There are extra config options! You can specify what artifacts it reports on. It defaults to `docs/build/index.html` but you can customise it i.e. https://github.com/matrix-org/matrix-doc/blob/master/pyproject.toml

Reply:
Fixed